### DISCUSSION TITLE
if hexo supports  mathematical formula of markdown or not?

### DISCUSSION THREAD

**Discussion by kirkzhangtech** [hexojs/hexo#5109] [Question / Answer] [answered]
![微信截图_20221120115011](https://user-images.githubusercontent.com/46363359/202883336-98ed6ffa-9e09-4703-be22-92cca93d838c.png)

hi  buddy
 i am not sure if this  issue caused by hexo or theme .  may i know  if hexo supports  mathematical formula of markdown or not?

**Comment by stevenjoezhang** [comment, marked as answer]
You need to install a plugin: https://github.com/next-theme/hexo-filter-mathjax or https://github.com/hexojs/hexo-math
